runtime/etc/debug/chpl_lldb_pretty_print.py

```diff
@@ -17,10 +17,67 @@
 # limitations under the License.
 #
 
+from functools import cache
 import lldb
 import re
 
 
+class Ptr:
+    def __init__(self, target, ptr, owned=True):
+        self.target = target
+        assert self.target is not None
+        self.ptr = ptr
+        self.owned = owned
+
+    def __del__(self):
```

Comment:
Something about `del` not necessarily being invoked when an object goes out of scope? Not sure if it matters in `lldb` anyway.

Reply:
It doesn't really matter here